tutorials/v7/text.cxx

```diff
@@ -37,21 +37,19 @@ void text()
 
    for (int i=0; i<=360; i+=10) {
       auto text = std::make_shared<Experimental::TText>(p, "____  Hello World");
```

Comment:
But then you don't need that `make_shared`! (That was the whole point of this exercise...) Just `Draw(TList(...))`

Reply:
Sure. Sorry for my slow brain ... :-(